### ISSUE TITLE
Passkey Support

### ISSUE THREAD

**Issue by phillmorgan28** [nextcloud/passman#811]
## Feature request

**User type**: Logged-in

**User level**: intermediate

### Description
Support for Passkeys

### Benefit / value
Modern security standard - vast majority of other managers now support them

### Risk / caveats
Unknown
 
**Are you a developer willing to implement this feature?**: no

**Comment by binsky08** [comment]
any idea how to achieve this?

**Comment by Fuseteam** [comment]
seems proton pass has achieved this, perhaps a hint can be found there?

**Comment by BuxarNET** [comment]
I'm also interested in Passkeys support. keepassxc supports it too, and they have browser extensions and open source code. Maybe you can get information there.

**Comment by morfran** [comment]
Yes, it would be a good addition. This project looks like a good place to start:

https://github.com/locke-inc/open-passkey
